spring-cloud-app-broker-acceptance-tests/src/test/java/org/springframework/cloud/appbroker/acceptance/fixtures/cf/UserCloudFoundryService.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2002-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.appbroker.acceptance.fixtures.cf;
+
+import java.util.Map;
+
+import org.cloudfoundry.operations.CloudFoundryOperations;
+import org.cloudfoundry.operations.DefaultCloudFoundryOperations;
+import org.cloudfoundry.operations.services.CreateServiceInstanceRequest;
+import org.cloudfoundry.operations.services.DeleteServiceInstanceRequest;
+import org.cloudfoundry.operations.services.GetServiceInstanceRequest;
+import org.cloudfoundry.operations.services.ServiceInstance;
+import org.cloudfoundry.operations.services.UpdateServiceInstanceRequest;
+import org.cloudfoundry.reactor.client.ReactorCloudFoundryClient;
+import org.cloudfoundry.reactor.tokenprovider.ClientCredentialsGrantTokenProvider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import reactor.core.publisher.Mono;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public final class UserCloudFoundryService {
+
+	private static final Logger LOG = LoggerFactory.getLogger(UserCloudFoundryService.class);
+
+	private static final String USER_CLIENT_ID = "app-broker-test-client";
```

Comment:
It's a bit surprising that these are here, along with the authorities a bit further down, but the equivalent configuration for `AdminCloudFoundryService` is in `CloudFoundryClientConfiguration`. Can we make this more consistent?

Reply:
Yep, sorry I missed that. Not sure I like having all these constants in `CloudFoundryClientConfiguration`, but I agree it's better to have it consitent.